Quests/Descriptions/quest_00050150.csv

```diff
@@ -1,5 +1,5 @@
 Title#0,"""Beach Wars!"""
-Explanation#0,"""夏だっ！\u3000海だっ！\u3000シーおねえさんだっ！<br>惑星ウォパルの海岸はまさに今が旬っ！<br>なのに暑さのせいか、凶暴化した海王種や<br>突然現れるようになったダーカーやらで<br>もー、バカンス計画がむっちゃくちゃ！<br>憩いの場を取り戻すためにも<br>アークスのみんなの力を貸して欲しいなっ！<br>さあ、まばゆいビーチを取り戻そうっ！<br>――――――――――――――――――――<br>Fail: Time limit expires"""
+Explanation#0,"""Summer's here! The beach is here! And big sis Xie is here too! Now is THE season for beach-going on Wopal! Yet, perhaps because of the heat, the Oceanids have gone wild, and Darkers have suddenly appeared, leaving our vacation plans in ruins! We need all ARKS to help take our oasis of relaxation back! So, come on, let's get this beautiful beach back!<br>――――――――――――――――――――<br>Fail: Time limit expires"""
```

Comment:
These descriptions have text limits and do require linebreaks, but we have to do this. It's why they weren't translated.

Reply:
I see. In that case, should I just revert the descriptions and leave the NPC text?

Reply:
We'll see able adding your translations in, but they won't be an exact match.